frontend/src/scenes/insights/filters/FunnelExclusionsFilter.tsx

```diff
@@ -98,8 +98,7 @@ function ExclusionRowSuffix({
             <div style={{ flex: 1 }} />
             <LemonButton
                 icon={<IconDelete />}
-                type="alt"
-                // size="small"
+                status="primary-alt" // size="small"
```

Comment:
we don't have the size="small" comment on other usages. Should we remove it here?

Reply:
Oh yeah thats some find and replace gone wrong